### ISSUE TITLE
data stream

### ISSUE THREAD

**Issue by ali-Eskandarian** [SeldonIO/alibi-detect#743]
Does this package support streaming data in anomaly detection?

**Comment by jklaise** [comment]
Our anomaly/outlier detectors can work on batches of arbitrary size as soon as the data is available, so in that sense yes, or did you have something else in mind?

**Comment by ali-Eskandarian** [comment]
no, I meant a stream of data that is constantly generating and I want to detect an anomaly as soon as it appears.

**Comment by jklaise** [comment]
The outlier detectors give a verdict on each data point in isolation, so yes, as you send data (perhaps even one-by-one) to a detector, you will get an immediate answer for it as there is no dependency on neighbouring points of data (as opposed to the case of offline drift-detectors that require a batch to be collected before making a verdict on the whole batch at once).
